src/error.rs

```diff
@@ -144,7 +141,6 @@ impl RuntimeError {
             DispatchError::CannotLookup => Ok(Self::CannotLookup),
             DispatchError::ConsumerRemaining => Ok(Self::ConsumerRemaining),
             DispatchError::NoProviders => Ok(Self::NoProviders),
-            DispatchError::TooManyConsumers => Ok(Self::TooManyConsumers),
```

Comment:
We may want to parameterize `DispatchError` and allow the definition in the metadata to be used, in case of such a mismatch in the `DispatchError` type between the target node and the released version of `sp_runtime`

Reply:
Working on that in https://github.com/paritytech/subxt/pull/394